README.md

```diff
@@ -37,7 +36,9 @@ To store objects in a local directory, use the following labels (example):
 "traefik.http.middlewares.my-aws.plugin.aws.directory" : "aws-local-directory"
 ```
 
-`GET` and `PUT` are supported.
+`GET`, `PUT` and `POST` are supported.
+`POST` will append a UUID to the path. There is an `ObjectLocation` header in the response.
```

Comment:
Unless there's a precedent for using `ObjectLocation` as the header name, I would just use `Location`, which is more standard.

Reply:
Sure! Object is the s3 terminology. I'll change it tho.